### DISCUSSION TITLE
Multiple types of animals in the same video?

### DISCUSSION THREAD

**Discussion by adam-matic** [talmolab/sleap#1227] [General]
Hi, I'm trying to make a program that tracks multiple types of animals in the video. Is it possible to train SLEAP to use multiple skeletons simultaneously? Or is it a better idea to train two networks, one for each type of animal? 
Thanks, 
Adam

**Comment by roomrys** [comment]
Hi @adadm-matic,

While SLEAP has some underlying structure set-up to grow in the direction of multi-species tracking within a single project, SLEAP does not currently have this feature. Currently, you will need to create two separate projects (one for each type of animal).

Thanks,
Liezl
